examples/sdkservice.go

```diff
@@ -1,34 +1,34 @@
 package main
 
 import (
-
+	"fmt"
 	"github.com/hpe-storage/nimble-golang-sdk/pkg/client/v1/nimbleos"
-	"github.com/hpe-storage/nimble-golang-sdk/pkg/param"
-	"github.com/hpe-storage/nimble-golang-sdk/pkg/service"
 	"github.com/hpe-storage/nimble-golang-sdk/pkg/fakeservice"
 	"github.com/hpe-storage/nimble-golang-sdk/pkg/sdkprovider"
-	"fmt"
+	"github.com/hpe-storage/nimble-golang-sdk/pkg/service"
+	"github.com/hpe-storage/nimble-golang-sdk/pkg/param"
 )
 
-func getFakeService(ip, username, password, apiVersion string, synchronous bool) (sdkprovider.NsGroupService, error) {
-
-	grpSvc, err := fakeservice.NewNsGroupService(ip, username, password, apiVersion, synchronous)
+func getFakeService(clientOpts ...service.ServiceOptions) (sdkprovider.NsGroupService, error) {
+	grpSvc, err := fakeservice.NewNimbleGroupService(clientOpts...)
 	return grpSvc, err
 }
 
-func getRealService(ip, username, password, apiVersion string, synchronous bool) (sdkprovider.NsGroupService, error) {
-	grpSvc, err := service.NewNsGroupService(ip, username, password, apiVersion, synchronous)
+func getRealService(clientOpts ...service.ServiceOptions) (sdkprovider.NsGroupService, error) {
+	grpSvc, err := service.NewNimbleGroupService(clientOpts...)
 	return grpSvc, err
 }
 
 func main() {
 	arg := &param.GetParams{}
-	groupService, _ := getFakeService("1.1.1.1", "xxx", "xxx", "v1", true)
+	groupService, _ := getFakeService(service.WithHost("1.1.1.1"),
+		service.WithUser("xxx"), service.WithPassword("xxx"),
+		service.WithoutWaitForAsyncJobs())
```

Comment:
Without -> With, or just remove this argument if it is the default.  Same thing at line 56.